incubator/java-microprofile/image/Dockerfile-stack

```diff
@@ -1,10 +1,20 @@
 FROM registry.access.redhat.com/ubi7/ubi
 
+LABEL vendor="Kabanero" \
+    name="Kabanero java-microprofile" \
+    version="0.1.0" \
```

Comment:
Is this the correct version number that we want?

Reply:
We should use the stack version, and change the name to kabanero/java-microprofile